integration/test/basic/basic_test.go

```diff
@@ -6,46 +6,18 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/giantswarm/e2e-harness/pkg/harness"
-	"github.com/giantswarm/helmclient"
-	"github.com/giantswarm/micrologger"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/helm/pkg/helm"
 )
 
 func TestInstallChart(t *testing.T) {
-	l, err := micrologger.New(micrologger.Config{})
-	if err != nil {
-		t.Fatalf("could not create logger %v", err)
-	}
-
-	config, err := clientcmd.BuildConfigFromFlags("", harness.DefaultKubeConfig)
-	if err != nil {
-		t.Fatalf("could not create k8s config %v", err)
-	}
-	cs, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		t.Fatalf("could not create k8s client %v", err)
-	}
-
-	c := helmclient.Config{
-		Logger:          l,
-		K8sClient:       cs,
-		RestConfig:      config,
-		TillerNamespace: "giantswarm",
-	}
-
-	helmClient, err := helmclient.New(c)
-	if err != nil {
-		t.Fatalf("could not create helm client %v", err)
-	}
```

Comment:
I want more tests here. So I'd like to simplify setup.

I want a test for IsReleaseAlreadyCreated (not introduced yet).
My goal is to make release installation idempotent in e2e-harness. We are refactoring things. Let's not take more debt and do things right.